src/Promenade.Web/Views/Home/LocationDetails.cshtml

```diff
@@ -0,0 +1,214 @@
+﻿@model Ocuda.Promenade.Controllers.ViewModels.LocationViewModel
+@{
+    ViewData["Title"] = "LocationDetails";
+}
+<html>
+<body>
+    <div id="content" class="location-page">
+        <img src="@Model.Location.ImagePath" class="branch-image" />
+        <h1>
+            @Model.Location.Name
+            <a href="@Model.Location.MapLink" class="btn btn-default map-button" target="_blank">
+                <span class="fas fa-map-marker-alt"></span> Map
+            </a>
+        </h1>
+        <table class="address-hours">
+            <tbody>
+                <tr>
+                    <td class="address">
+                        <p>
+                            @Model.Location.Address
+                            <br />
+                            @Model.Location.City, AZ @Model.Location.Zip
+                            <br />
+                            <a href="tel:+1 @Model.Location.Phone">@Model.Location.Phone</a>
+                        </p>
+                    </td>
+                    <td class="hours">
+                        <strong>Mon. - Sun.</strong> Closed<br />
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h2>Information</h2>
+        <p>@Model.Location.Description</p>
+        <div>
+            <div>
+                <a href="@Model.Location.EventLinkId" target="_blank">Browse Events</a>
+            </div>
+            <div>
+                <a href="@Model.Location.SubscriptionLinkId" target="_blank">Send Me Events</a>
+            </div>
+        </div>
+        <div class="row branch-features text-center">
+            @foreach (var feature in Model.LocationFeatures)
+            {
+                <text>
+                    <div class="col-xs-2">
+                        @if (feature.RedirectUrl != null)
+                        {
+                            <a href="@feature.RedirectUrl" target="_blank">
+                                <span class="fa-stack fa-2x" style="color:#007bff">
+                                    <span class="fa fa-square fa-stack-2x"></span>
+                                    <span class="@feature.FontAwesome">
+                                        @if (feature.Name == "7-Day Express")
```

Comment:
This "7-Day Express" exception should not be in the view, we need to figure out how to properly display this feature using information from the database.